Assignment1/src/main/java/models/Deck.java

```diff
@@ -9,21 +9,52 @@
  * Deck Class that uses the Card class to maintain a sinlge unique deck.
  */
 public class Deck {
-    private ArrayList<Card> deckList;
+    private static int NUM_SIZE = 52;
+    public Card cards[];
+    public int count;
+
+    public ArrayList<Card> deckList;
```

Comment:
Is this needed anymore? I don't see it being used anywhere else. It looks like cards[] replaces this